app/src/main/java/org/jellyfin/androidtv/ui/playback/MediaManager.java

```diff
@@ -305,40 +306,40 @@ public void onAudioFocusChange(int focusChange) {
         }
     };
 
-    private static void fireQueueReplaced(){
+    private void fireQueueReplaced(){
         for (AudioEventListener listener : mAudioEventListeners) {
             Timber.i("Firing queue replaced listener. ");
             listener.onQueueReplaced();
         }
     }
 
-    private static void fireQueueStatusChange() {
+    private void fireQueueStatusChange() {
         for (AudioEventListener listener : mAudioEventListeners) {
             Timber.i("Firing queue state change listener. %b", hasAudioQueueItems());
             listener.onQueueStatusChanged(hasAudioQueueItems());
         }
 
     }
 
-    private static void createAudioQueue(List<BaseItemDto> items) {
+    private void createAudioQueue(List<BaseItemDto> items) {
         mCurrentAudioQueue = new ItemRowAdapter(items, new CardPresenter(true, Utils.convertDpToPixel(TvApp.getApplication(), 140)), null, QueryType.StaticAudioQueueItems);
         mCurrentAudioQueue.Retrieve();
         mManagedAudioQueue = null;
         fireQueueStatusChange();
     }
 
-    private static int TYPE_AUDIO = 0;
-    private static int TYPE_VIDEO = 1;
+    private int TYPE_AUDIO = 0;
+    private int TYPE_VIDEO = 1;
```

Comment:
These should probably be `static` and `final`.

Reply:
Good catch